test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -266,83 +272,459 @@ var _ = g.Describe("Scylla Manager integration", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(tasks.TaskListItemSlice).To(o.BeEmpty())
 	})
+})
+
+var _ = g.Describe("Scylla Manager integration (requiring object storage)", framework.RequiresObjectStorage, func() {
```

Comment:
This could actually live in a separate file `scyllamanager_extended.go` or something like that.

Reply:
moved to `scyllamanager_object_storage` - extended feels wrong as this is a core functionality, just has an additional requirement to run